client/ayon_core/hosts/houdini/plugins/create/create_vray_rop.py

```diff
@@ -143,20 +147,41 @@ def remove_instances(self, instances):
 
         return super(CreateVrayROP, self).remove_instances(instances)
 
+    def get_instance_attr_defs(self):
+        """get instance attribute definitions.
+
+        Attributes defined in this method are exposed in
+            publish tab in the publisher UI.
+        """
+
+
+        render_target_items = {
+            "local": "Local machine rendering",
+            "local_no_render": "Use existing frames (local)",
+            "farm": "Farm Rendering",
+            "farm_split": "Farm Rendering - Split export & render jobs",
+        }
+
+        return [
+            BoolDef("review",
+                    label="Review",
+                    tooltip="Mark as reviewable",
+                    default=True),
+            EnumDef("render_target",
+                    items=render_target_items,
+                    label="Render target",
+                    default=self.render_target)
+        ]
```

Comment:
Can we please make a follow-up PR that abstract this to a parent class so this logic is basically re-used for all render creators in Houdini?

This PR clearly shows that basically all these creators share 80% of logic between each other? Can't we unify this in a parent "CreateRender" class?

Reply:
We should indeed, tagging @MustafaJafar

Reply:
I've created an issue to track it. Thank you!
https://github.com/ynput/ayon-core/issues/507